### DISCUSSION TITLE
Loupedeck+ [Plus] support?

### DISCUSSION THREAD

**Discussion by Tridens92** [calibx/msfsdeck#185] [Ideas]
I'm new here, and was excited to see this project. Unfortunately, I only own a Loupedeck+ (the 2nd generation without any LCD buttons).  But with all its' buttons, rotary dials, and sliders, it would be a great control unit for MSFS.  Any chance this unit could get integrated into the project?  I would assume the API is very similar.

Here's hoping and praying! :)

**Comment by calibx** [comment]
Hi,

Sorry but I don't know if the SDK work with the + version.
In the SDK sample it is not covered (for example the profiles) : https://github.com/Loupedeck/PluginSdk/wiki/Default-Application-Profiles
I asked the SDK leader about that. I hope to have a better answer from him.

**Comment by Tridens92** [comment]
Great, @calibx !  Thanks so much for checking on that.

**Comment by calibx** [comment]
The + will be included in the 5.4 SDK so I will be able to try that !
